README.md

```diff
@@ -33,4 +33,27 @@ Or, you may add these entries maually using your editor of choice:
     <key>NSBluetoothPeripheralUsageDescription</key>
     <string>Please enable bluetooth to continue.</string>
 </dict>
+```
+
+## Using
+
+### Searching for nearby devices
+
+```dart
+Map<String, ScanResult> results = <String, ScanResult>{};
+
+Stream scanStream = RxBle.startScan();
+StreamSubscription scanSubscription;
+
+scanSubscription = scanStream.listen((scanResult) async {
+    if (scanResult.deviceName != null) {
```

Comment:
Did you encounter a situation where `deviceName` was `null`, that you felt a need to add this check?

Reply:
> Why did you delete all the `lock` files?

I deleted `lock` files because my changes are only in the documentation not in `lock` files.

> Did you encounter a situation where `deviceName` was `null`, that you felt a need to add this check?

Yes, I found many devices with their names null.

Reply:
You deleting the `lock` files also deletes them on the master!

Reply:
> You deleting the `lock` files also deletes them on the master!

I'm sorry, it was my first time contributing in a open source repository, I will fix this and update my pull request, thanks for your report.

Reply:
Hmm, maybe it will be easier for you to clone again and create a new PR :)

Or just use the online editor on GitHub, since this is only a `README.md` change.

Reply:
Ok, I will do it!